src/components/screens/HomeScreen/WelcomeBannerBottomSheet.tsx

```diff
@@ -39,7 +39,6 @@ const CustomContent: React.FC<{
           {t("welcomeBanner.welcomeMessage")}
         </Text>
         <View className="h-4" />
-        <View className="h-px mb-4 bg-border-primary" />
```

Comment:
Removes a border that was here. This now matches the Figma:

<img width="445" height="965" alt="Screenshot 2025-10-29 at 5 01 24 PM" src="https://github.com/user-attachments/assets/6ff613c8-9ad7-4d67-8dd1-1e286212886d" />